### PR TITLE
Day4

### PR DESCRIPTION
**Pull request by andreealung96**

### CODE REVIEW

File: Proiect-VS/assets/css/evaluations.less [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    color: #686865;
+    text-align: right;
+    padding-right: 50px;
+
+}
+
+a {
+    vertical-align: middle;
+    line-height: 70px;
+    margin-left: 100px;
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/evaluations.less:29]: these are not generic enought styles to be applied to all a elements( eg margin-left:100px etc). Please use a class to apply these margings and keep in "a" only the truly generic properties.

File: Proiect-VS/assets/css/evaluations.less [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    background-color: #ffffff; 
+    margin: auto;
+}
+
+header {
+    height: 70px;
+    border-bottom: 1px solid grey;
+}
+
+footer {
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/evaluations.less:13]: I would use a class

File: Proiect-VS/assets/css/evaluations.less [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    height: 60px; 
+    position: absolute;
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/evaluations.less:15]: Please do not position using absolute positioning

File: Proiect-VS/assets/css/evaluations.less [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    left: 0;
+    right: 0;
+    line-height: 60px;
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/evaluations.less:18]: break down this rule into many small classes.

File: Proiect-VS/assets/css/evaluations.less [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    margin: 200px auto 440px;
+    width: 60%;
+    height: 200px;
+    border-collapse: collapse;
+    text-align: left;
+    font-size: 20px;
+}
+
+.candidates_table_header {
+    background-color: #21bcff;
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/evaluations.less:51]: Define a colors.less files and define all of the colors as vars in there. Remove the hardcoded color values.

File: Proiect-VS/assets/css/evaluations.less [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+    border: none;
+    height: 60px;
+    color: white;
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/evaluations.less:54]: color is skin. separate it out into it's own class.

File: Proiect-VS/assets/css/evaluations.less [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+    padding-left: 30px;
+}
+
+.current_page {
+    border-bottom: 9px solid #21bcff;
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/evaluations.less:59]: border is skin

File: Proiect-VS/assets/css/evaluations.less [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+.rows {
+    border-bottom: 1px solid #686865;
+    height: 40px;
+}
+
+button {
+    border: none;
+    border-radius: 100%;
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/evaluations.less:70]: border, border-radius etc is skin

File: Proiect-VS/assets/css/login.less [hunk 9]
```diff
@@ -0,0 +1,116 @@
+body {
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/login.less:1]: i would define and use a generic, utility class .center {}

File: Proiect-VS/assets/css/login.less [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+
+.skin_login {
+     background-color: white;
+     box-shadow: 0px 2px 2px #333435;   
+}
+
+.login_header {    
+    position: absolute; 
+    width:100%;
+    border-radius: 3px;
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/login.less:32]: border-radius is skin

File: Proiect-VS/assets/css/login.less [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+    height: 40px;
+}
+
+.skin_login_header {
+    background-color: #419fe2; 
+    color: white;
+    font-size: 20px;
+}
+
+.login_input {
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/login.less:46]: General feedback: in general '-' convention is used in CSS( eg login-input etc )

File: Proiect-VS/assets/css/login.less [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+    border: none;
+    border-bottom: 0.5px solid grey;
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/login.less:48]: this is skin

File: Proiect-VS/assets/css/login.less [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+    border: none;    
+    padding: 5px 30px;
+    width: 35%;
+    margin-left: 95px;
+    border-radius: 3px;    
+}
+
+.skin_login_submit {
+    background-color: #419fe2;
+    color: white;
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/login.less:64]: Check if the font/color combination isn't used anywhere else in which case extract it into a standalone class for re-usability

File: Proiect-VS/assets/css/login.less [hunk 14]
```diff
@@ -0,0 +1,116 @@
...
+    margin-left: 30px;
+    margin-right: 10px;
+}
+
+.login_submit {    
+    border: none;    
+    padding: 5px 30px;
+    width: 35%;
+    margin-left: 95px;
+    border-radius: 3px;
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/login.less:59]: border/border-radius is skin

File: Proiect-VS/assets/css/login.less [hunk 15]
```diff
@@ -0,0 +1,116 @@
...
+.section1 {
+    height: 350px;
+    align-content: center;
+    padding-top: 5px;
+    padding-bottom: 15px;
+    padding-left: 500px;
+    padding-right: 500px;
+}
+
+#img1 {
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/login.less:78]: no id selectors

File: Proiect-VS/assets/css/login.less [hunk 16]
```diff
@@ -0,0 +1,116 @@
...
+    height: 250px;
+    align-content: center;
+    padding-top: 50px;
+    padding-bottom: 50px;
+    padding-left: 500px;
+    padding-right: 500px;
+    background-color: #f2f4f1;
+}
+
+#img2 {
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/login.less:95]: no id selector

File: Proiect-VS/assets/css/new-evaluation.less [hunk 17]
```diff
@@ -0,0 +1,162 @@
...
+    font-family: Arial;
+}
+
+header {
+    height: 70px;
+    border-bottom: 1px solid grey;
+    font-size: 25px;
+}
+        
+footer {
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/new-evaluation.less:17]: use a class. Breakdown the selector into smaller classes.

File: Proiect-VS/assets/css/new-evaluation.less [hunk 18]
```diff
@@ -0,0 +1,162 @@
...
+    border-bottom: 3px solid #f7f7f7;
+    height: 30px;
+    width: 300px;
+    margin: 7px;
+}
+
+.evaluation_form_table {
+    border-collapse: collapse;
+    width: 1000px;
+    margin: 20.750px 0px;
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/new-evaluation.less:77]: 20.750px is really weird. use a more round measurement

File: Proiect-VS/assets/css/new-evaluation.less [hunk 19]
```diff
@@ -0,0 +1,162 @@
...
+    height: 40px;
+    width: 934px;
+}
+
+.evaluation_form_submit {
+    float: right;
+    width: 150px;
+    height: 50px;
+    border: none;
+    border-radius: 2px;
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/new-evaluation.less:103]: a little skin in here

File: Proiect-VS/assets/css/new-evaluation.less [hunk 20]
```diff
@@ -0,0 +1,162 @@
...
+    margin-bottom: 10px;
+    margin-top: 10px;
+}
+
+@media only screen and (max-device-width: 480px) {
+      .evaluation_form_submit {text-transform: uppercase; }
+}
+
+.skin_evaluation_form_submit {
+    background-color: #21bcff;
```
Comment (gcazaciuc) [inline Proiect-VS/assets/css/new-evaluation.less:114]: good job at doing this separation. You should also check if some of these props combinations aren't used elsewhere in which case you can further abstract them

File: Proiect-VS/evaluations.html [hunk 21]
```diff
@@ -0,0 +1,53 @@
...
+	<head>
+		<title>Evaluations</title>
+		<link rel="stylesheet/css" type="text/css" href="assets/css/normalize.css" />
+        <link rel="stylesheet/less" type="text/css" href="assets/css/evaluations.less">
+        <script src="less.js" type="text/javascript"></script>
+	</head>
+	<body>
+		<header>
+			<img class="logo" src="assets/logo-v2.png">
+			<a class="current_page" href="evaluations.html">Evaluations</a>
```
Comment (gcazaciuc) [inline Proiect-VS/evaluations.html:12]: the links should probably be in a nav

File: Proiect-VS/login.html [hunk 22]
```diff
@@ -0,0 +1,53 @@
...
+	<link rel="stylesheet/less" type="text/css" href="assets/css/login.less">
+	<script src="less.js" type="text/javascript"></script>
+	</head>
+	<body class="skin_body">
+		<header>
+			<img class="banner" src="assets/feedback-banner.jpg">
+			<div class="login skin_login">
+				<div class="login_header skin_login_header">Interview Feedback</div>
+				<form action="" method="post">
+					<input class="login_input" style="margin-top: 90px" type="text" name="username" placeholder="Username"><p>
```
Comment (gcazaciuc) [inline Proiect-VS/login.html:15]: remove inline styles

File: Proiect-VS/login.html [hunk 23]
```diff
@@ -0,0 +1,53 @@
...
+					<input class="login_input" style="margin-bottom: 45px" type="password" name="password" placeholder="Password"><p>
+					<button class="login_submit skin_login_submit" type="Submit">Login</button>
+				</form>
+			</div>
+		</header>
+		<section class="section1">
+			<img id="img1" src="assets/new feedback.jpg">
```
Comment (gcazaciuc) [inline Proiect-VS/login.html:22]: set alt atttibute

File: Proiect-VS/login.html [hunk 24]
```diff
@@ -0,0 +1,53 @@
...
+			<h3 style="font-family: Arial" >Submit new feedback</h3>
```
Comment (gcazaciuc) [inline Proiect-VS/login.html:23]: remove inline and use classes everywhere

File: evaluations.css [hunk 25]
```diff
@@ -0,0 +1,97 @@
+		body {
```
Comment (gcazaciuc) [inline evaluations.css:1]: Convert to .less, fix indentation and use classes

File: new-evaluation.css [hunk 26]
```diff
@@ -1,18 +1,55 @@
...
 		}
-		
-		.header {
+		header {
 			height: 70px;
 			border-bottom: 1px solid grey;
 		}
+				
+		footer {
+			height: 60px;
```
Comment (gcazaciuc) [inline new-evaluation.css:15]: use a class, split out the selector into smaller classes

File: new-evaluation.css [hunk 27]
```diff
@@ -21,60 +58,57 @@
...
-		}
-
-		a:visited {
-			text-decoration: none;
-			color: black;
+		.evaluation_form_submit {
+			float: right;
+			width: 150px;
+			height: 50px;
+			background-color: #21bcff;
```
Comment (gcazaciuc) [inline new-evaluation.css:101]: structure/skin

File: new-evaluation.css [hunk 28]
```diff
@@ -140,4 +144,15 @@
...
\ No newline at end of file
+		}
+
+		legend {
+  			margin: 10px 20px;
+  			
+  		}
+
+  		@media only screen and (max-device-width: 480px) {
+			.evaluation_form_submit {text-transform: uppercase; }
```
Comment (gcazaciuc) [inline new-evaluation.css:155]: this rule doesn't make much sense to be applied only when screen <= 480

Comment (gcazaciuc) [reply new-evaluation.css:155]: Remove if not needed anymore

File: new-evaluation.html [hunk 29]
```diff
@@ -4,27 +4,29 @@
...
-	<body style="background-color: #f7f7f7; margin: auto">
-		<header class="header">
-			<img id="logo" src="logo-v2.png">
-			<a style="vertical-align: middle; line-height: 70px" href="evaluations.html">Evaluations</a>
-			<a style="border-bottom: 9px solid #419fe2;vertical-align: middle; line-height: 70px; padding-bottom: 12px" href="new-evaluation.html">New Evaluation</a>
-			<a style="float: right; vertical-align: middle; line-height: 70px" href="login.html">Logout</a>
+	<body>
+		<header>
+			<img id="logo" src="assets/logo-v2.png">
+			<a href="evaluations.html">Evaluations</a>
```
Comment (gcazaciuc) [inline new-evaluation.html:10]: maybe wrap them in a nav(the links) ?

File: new-evaluation.html [hunk 30]
```diff
@@ -4,27 +4,29 @@
...
+			<a class="current_page" href="new-evaluation.html".>New Evaluation</a>
+			<a href="login.html" style="float: right">Logout</a>
```
Comment (gcazaciuc) [inline new-evaluation.html:12]: remove inline style

File: new-evaluation.html [hunk 31]
```diff
@@ -4,27 +4,29 @@
...
 		</header>
-			<div class="div_form">
-				<form method="post" action="">
-					<input class="input" type="text"  name="Candidate" placeholder="Candidate">
-					<input class="input" type="text" name="Interviewer" placeholder="Interviewer">
-					<input class="input" type="date" name="Date" value="04/22/2012">
+			<div class="evaluation_form">
+				<form method="post" action="action.php">
```
Comment (gcazaciuc) [inline new-evaluation.html:15]: action.php ? :)

**Review by gcazaciuc — APPROVED** [review]

File: evaluations.css (hunk 25 above)
Comment (andreealung96) [reply evaluations.css:1]: This is an older version of the file, don't know why I uploaded it. By now, I've already made a .less file.

File: new-evaluation.css (hunk 28 above)
Comment (andreealung96) [reply new-evaluation.css:155]: I used this media query cause Alex asked me if I was able to make the submit button be displayed in capital case on screen, and in uppercase only on mobile devices.

Comment (softvision-alexcarcea) [reply new-evaluation.css:155]: Yep, I did, and she proved she could. What more can one ask for? :smile:

**Pull request merged**